category-committed.php

```diff
@@ -0,0 +1,330 @@
+          <?php the_content(); ?>
+        </div>
+      </article>
+    <?php
+  }
+}
+?>
+    </div>
+  </section>
+  <div>
+    <div class="committed-credits__background pb-6 pt-6 font-color-white">
+      <div class="flex-grid-row mb-6 mb-s-5 mt-m-0 mt-4 committed-credits__logo-row">
+        <img src="<?php echo $base_image_path . 'committed-logo-white.png'; ?>" alt="committed credits logo" class="committed-credits__logo"/>
+      </div>
+      <div class="flex-grid-row committed-credits__info-row font-size-10">
+
+        <!-- Reporters -->
+        <div class="flex-grid-item flex-item-m-4 flex-item-xxl-4 flex-item-s-8 mb-s-5">
+          <div class="flex-grid-row font-weight-bold mb-1">Reporters</div>
+          <div class="committed-credits__reporters flex-grid-row">
+            <div class="mr-2">
+              <a href="<?php echo home_url( 'contributor/rivkah-brown/' ); ?>" class="ui-hover">
+              <img src="<?php echo $base_image_path . 'committed-rivkah.jpg'; ?>" alt="Portrait of Rivkah Brown" class="committed-credits__logo"/>
+              </a>
+              <p>Rivkah Brown</p>
+            </div>
+            <div class="">
+              <a href=<?php echo home_url( 'contributor/clare-hymer/' ); ?> class="ui-hover">
+              <img src="<?php echo $base_image_path . 'committed-clare.jpg'; ?>" alt="Portrait of Clare Hymer" class="committed-credits__logo"/>
+              </a>
+              <p>Clare Hymer</p>
+            </div>
+          </div>
+        </div>
+
+        <!-- Producers -->
+        <div class="flex-grid-item flex-item-m-4 flex-item-l-3 flex-item-xxl-3 flex-item-s-8 mb-s-5">
+          <div class="font-weight-bold mb-1">Producers</div>
+          <div class="mb-2">
+            <?php echo wp_kses_post( $credits['producers'] ); ?>
+          </div>
+          <div class="font-weight-bold mb-1">Music and sound editing</div>
+          <div class="mb-2">
+            <?php echo wp_kses_post( $credits['music_sound'] ); ?>
+          </div>
+          <div class="font-weight-bold mb-1">Design and digital</div>
+          <div class="mb-2">
+            <?php echo wp_kses_post( $credits['design_digital'] ); ?>
+          </div>
+          <div class="font-weight-bold mb-1">Social media production</div>
+          <div>
+            <?php echo wp_kses_post( $credits['social_media'] ); ?>
+          </div>
+        </div>
+
+        <div class="flex-grid-item flex-item-m-8 flex-item-l-4 flex-item-xxl-3 mt-m-5 mt-s-0 flex-item-s-8">
+          <div class="font-weight-bold mb-1">Thanks to</div>
+          <?php echo wp_kses_post( $credits['thanks'] ); ?>
+
+          <div class="font-weight-bold mt-5 mb-1">Image credits</div>
+          <?php echo wp_kses_post( $credits['image_covers'] ); ?>
```

Comment:
No spaces or line breaks for these credits? Probably the headings here should be h5s or h6s as that is more semantic, but not sure exactly how that would stack down the flow of the page

Reply:
Added line breaks and swopped to h6